### PR TITLE
Remove tautology from response section

### LINKED ISSUE #98: Remove tautology from section 6
Section 6 contains the following paragraph:
> A VP Token is only returned if the corresponding Authorization Request contained a presentation_definition parameter, a presentation_definition_uri parameter, or a scope parameter representing a Presentation Definition [Section 5](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#vp_token_request).

This is redundant since if this is not the case, an error is returned anyways.

### PR DESCRIPTION
**Pull request by awoie**
This PR does:
- Removes tautology from response section
- Fixes #98 

IMO, this PR is non-breaking and editorial.

### CODE REVIEW

**Review by Sakurann — APPROVED** [review]
a bulletpoint in https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-6.4-5.1 says the same thing, so probably ok to remove..

File: openid-4-verifiable-presentations-1_0.md [hunk 1]
```diff
@@ -462,9 +462,7 @@ Other specifications can define further values for the `client_id_scheme` parame
 
 # Response {#response}
 
-A VP Token is only returned if the corresponding Authorization Request contained a `presentation_definition` parameter, a `presentation_definition_uri` parameter, or a `scope` parameter representing a Presentation Definition (#vp_token_request).
```
Comment (jogu) [inline openid-4-verifiable-presentations-1_0.md:465]: I'm not really convinced that this sentence is unnecessary, and in fact I'd more on the side of thinking it's helpful. If others are convinced it is superfluous then fine, I'll not block this, but I thought it was worth stating why I've not added a positive review.

Comment (Sakurann) [reply openid-4-verifiable-presentations-1_0.md:465]: yeah, honestly, I am not super convinced it is redundant either..

Comment (awoie) [reply openid-4-verifiable-presentations-1_0.md:465]: We can close this PR if you think it is not redundant which I still think it is.

**Comment by awoie** [comment]
Closing this PR since no consensus.

**Pull request closed without merging**